docs/root/start/sandboxes/route_mirroring.rst

```diff
@@ -0,0 +1,126 @@
+.. _install_sandboxes_route_mirroring:
+
+Route mirroring policies
+========================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+This simple example demonstrates Envoy's request mirroring capability using
+`request mirror policies <https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-routeaction-requestmirrorpolicy>`__.
+
+Incoming requests are received by ``front-envoy`` Envoy proxy.
+
+The ``front-envoy.yaml`` envoy configuration configures two routes::
+
+        routes:
+          - match:
+              prefix: "/service/1"
+            route:
+              cluster: service1
+              request_mirror_policies:
+                cluster: "service1-mirror"
+          - match:
+              prefix: "/service/2"
+            route:
+              cluster: service2
+              request_mirror_policies:
+                cluster_header: "x-mirror-cluster"
+
+A request for the path ``/service/1`` is forwarded to the ``service1`` cluster.
+In addition, the request is also forwarded to the ``service1-mirror`` cluster.
+
+A request for the path ``/service/2`` is forwarded to the ``service2`` cluster.
+If a header ``x-mirror-cluster`` is specified in the request, envoy extracts the
+header value and forwards the request to a cluster with the same name, if found.
+For example, if we send a header with a request as ``x-mirror-cluster: service2-mirror``,
+the request will be forwarded to the ``service2-mirror`` cluster.
+
+Step 1: Build the sandbox
+*************************
+
+Change to the ``examples/route-mirroring`` directory.
+
+Terminal 1 ::
+
+    $ pwd
+    envoy/examples/route-mirroring
+    $ docker-compose build
+    $ docker-compose up
+
+
+Terminal 2 ::
```

Comment:
im not understanding why this is separated here

Reply:
Do you mean, why we have this in a separate terminal here? When we run `docker-compose up` in Terminal 1, we are not able to execute any more commands as we are not using the detached mode `-d` for `docker-compose`.

Reply:
i think rather follow the same pattern that other sandboxes use of brining up the compose services with `-d` and then there is no need for multiple terminals

Reply:
@phlax i agree that it may be a good idea in general. However, for this sandbox, viewing the logs from the application is the most straightforward to see that the request was also sent to the mirror cluster. Hence, having the separate terminal open is IMHO better than running `docker logs`. 

If you prefer `docker logs` instead of having the terminal open, happy to change it.

Reply:
yes - please make this consistent with how the other sandboxes work

Reply:
https://github.com/envoyproxy/envoy/pull/23336/commits/3621677290368190a909ca7d960b520088cb155c